README.md

```diff
@@ -61,13 +61,14 @@ Here's an example of a CDC position:
-| `resource`      | The HubSpot resource that the connector will work with.<br />You can find a list of the available resources [here](docs/resources.md). | **true** |         |
-| `maxRetries`    | The number of HubSpot API request retries attempts that will be tried before giving up if a request fails.                             | false    | `4`     |
-| `pollingPeriod` | The duration that defines a period of polling new items.                                                                               | false    | `5s`    |
-| `bufferSize`    | The buffer size for consumed items.<br />It will also be used as a limit when retrieving items from the HubSpot API.                   | false    | `100`   |
+| name              | description                                                                                                                                                                                                                           | required | default |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- |
+| `accessToken`     | The private app access token for accessing the HubSpot API.                                                                                                                                                                           | **true** |         |
+| `resource`        | The HubSpot resource that the connector will work with.<br />You can find a list of the available resources [here](docs/resources.md).                                                                                                | **true** |         |
+| `maxRetries`      | The number of HubSpot API request retries attempts that will be tried before giving up if a request fails.                                                                                                                            | false    | `4`     |
+| `pollingPeriod`   | The duration that defines a period of polling new items.                                                                                                                                                                              | false    | `5s`    |
+| `bufferSize`      | The buffer size for consumed items.<br />It will also be used as a limit when retrieving items from the HubSpot API.                                                                                                                  | false    | `100`   |
+| `extraProperties` | The list of HubSpot resource properties to include in addition to the default.<br />If any of the specified properties are not present on the requested HubSpot resource, they will be ignored.<br />Only CRM resources support this. | false    |         |
```

Comment:
maybe we can add some example how it looks, something like:
`body,name`

Reply:
I've added an example of how it looks